cli/tests/test_global_opts.rs

```diff
@@ -559,6 +559,16 @@ fn test_early_args() {
     let stdout = test_env.jj_cmd_success(test_env.env_root(), &["help", "--color=always"]);
     insta::assert_snapshot!(stdout.lines().find(|l| l.contains("Commands:")).unwrap(), @"[1m[4mCommands:[0m");
 
+    // Check that early args are accepted after -h/--help
```

Comment:
(Optional) Should we also test `--color=always -h`?Or is that completely equivalent to `--color=always --help`?

Reply:
They're equivalent in implementation wise.